x11-libs/libxfixes/libxfixes-5.0.3.recipe

```diff
@@ -66,8 +65,7 @@ defineDebugInfoPackage libxfixes$secondaryArchSuffix \
 
 BUILD()
 {
-	libtoolize --force --copy --install
-	autoreconf -i
+	autoreconf -vfi
```

Comment:
Should these be needed, in this case libxfixes builds fine without "autoreconf -vfi" and I'm pretty sure it's not needed for the other x recipes?

Reply:
Nevermind, looks like in some cases it's needed :)